webapi/Controllers/ChatHistoryController.cs

```diff
@@ -122,22 +129,25 @@ public async Task<IActionResult> GetChatSessionByIdAsync(Guid chatId)
     }
 
     /// <summary>
-    /// Get all chat sessions associated with a user. Return an empty list if no chats are found.
-    /// The regex pattern that is used to match the user id will match the following format:
-    ///    - 2 period separated groups of one or more hyphen-delimited alphanumeric strings.
-    /// The pattern matches two GUIDs in canonical textual representation separated by a period.
+    /// Get all chat sessions associated with the logged in user. Return an empty list if no chats are found.
     /// </summary>
-    /// <param name="userId">The user id.</param>
     /// <returns>A list of chat sessions. An empty list if the user is not in any chat session.</returns>
     [HttpGet]
     [Route("chatSession/getAllChats/{userId:regex(([[a-z0-9]]+-)+[[a-z0-9]]+\\.([[a-z0-9]]+-)+[[a-z0-9]]+)}")]
```

Comment:
This regex will fail to match in the no auth case since the userId is a single GUID instead of user.tenant. I think we can just remove the regex contraint for now and allow the {userId} to be any value since we can ignore it anyway.